src/main/webapp/dashboard.js

```diff
@@ -104,11 +147,109 @@ function createLocation(location) {
+  AFFILIATIONS_DIV.append(projectCardElement);
+  let menteeRow = createRowElement();
+  for (mentee of projectObject.mentees) {
+    menteeRow.append(createMenteeCard(mentee));
+  }
+  projectCardElement.append(menteeRow);
+}
+
+/**
+ * Return a project card div element with the title @param projectName
+ * @param {String} projectName
+ * @return {HTMLElement} div representing a project card with the
+ * title @param projectName
+ */
+function createProjectCard(projectName) {
+  let projectCardElement = document.createElement('div');
+  projectCardElement.className = 'card container card-holder col-12' +
+    ' text-center project-card p-3 m-3';
+  let projectTitleElement = createCardTitle(projectName);
+  projectCardElement.append(projectTitleElement);
+  return projectCardElement;
+}
+
+/**
+ * Return a bootstrap card-title div element with @param titleText
+ * @param {String} titleText
+ * @return {HTMLElement} div representing a card title with @param titleText
+ */
+function createCardTitle(titleText) {
+  let cardTitleElement = document.createElement('h4');
+  cardTitleElement.className = 'card-title dark-emph';
+  cardTitleElement.innerText = titleText;
+  return cardTitleElement;
+}
+
+/**
+ * Return a card with information about a given @param mentee object and
+ * a link to send @param mentee a star
+ * @param {Mentee} mentee
+ * @return {HTMLElement} card with information about a given mentee
+ */
+function createMenteeCard(mentee) {
+  let menteeCardElement = document.createElement('div');
+  menteeCardElement.className = 'card card-holder col-' +
+    ' text-center m-1 p-2';
+  let menteeCardBody = document.createElement('div');
+  menteeCardBody.className = 'card-body';
+  menteeCardElement.append(menteeCardBody);
+  menteeCardBody.append(createMenteeCardImage(mentee.image));
+  menteeCardBody.append(createCardTitle(mentee.name));
+  menteeCardBody.append(createStarButton(mentee.starLink));
+  return menteeCardElement;
+}
+
+/**
+ * Return a bootstrap row element
+ * @return {HTMLElement} bootstrap row div
+ */
+function createRowElement() {
+  let rowElement = document.createElement('div');
+  rowElement.className = 'row p-3';
+  return rowElement;
+}
+
+/**
+ * Return an anchor of the class star that perform the acton @param link
+ * @param {String} link
+ * @return {HTMLElement} anchor that performs the given action
+ */
+function createStarButton(link) {
+  let starButton = document.createElement('a');
+  starButton.className = 'star';
+  starButton.innerText = 'Send star';
+  starButton.href = link;
+  return starButton;
+}
+
+/**
+ * Return a small image with the source @param imgSrc
+ * @param {String} imgSrc
+ * @return {HTMLElement} image object
+ */
+function createMenteeCardImage(imgSrc) {
+  let menteeCardImage = document.createElement('img');
+  menteeCardImage.className = 'card-img-top mentee-card-img';
+  menteeCardImage.src = imgSrc;
+  return menteeCardImage;
+}
+
 /**
```

Comment:
I am loving how you broke this up into many small, simple, logical functions!

Reply:
Thanks!